sway/input/seatop_move_floating.c

```diff
@@ -6,6 +6,7 @@
 
 struct seatop_move_floating_event {
 	struct sway_container *con;
+	double ox, oy; // cursor offset in container
```

Comment:
Could add "when seatop started" to make it clear it's not the current offset. Also we generally use `ox`/`oy` for output-local coords, so maybe we should rename that to `cx`/`cy` (container) or `dx`/`dy` (delta).

Reply:
Well the whole point of the patch is that the offset stays the same, even if you try to move the container somewhere where the pointer is constrained, so I think it makes sense as is.

I see why `ox, oy` are bad names, noted. I hadn't made the connection to output-local coords. I guess I'll use dx/dy here then.

Reply:
>Well the whole point of the patch is that the offset stays the same, even if you try to move the container somewhere where the pointer is constrained, so I think it makes sense as is.

Eh, indeed!